decidim-elections/app/controllers/decidim/elections/elections_controller.rb

```diff
@@ -0,0 +1,62 @@
+# frozen_string_literal: true
+
+module Decidim
+  module Elections
+    class ElectionsController < ApplicationController
+      include Decidim::ApplicationHelper
+      include Decidim::AttachmentsHelper
+      include Decidim::FilterResource
+      include Decidim::Elections::Orderable
+      include Decidim::Paginable
+
+      helper_method :elections, :election, :tab_panel_items, :questions, :paginated_elections
+
+      def index; end
+
+      def show
+        raise ActionController::RoutingError, "Not Found" unless election
+
+        respond_to do |format|
+          format.html { render :show }
+
+          format.json do
+            render json: election.to_json
+          end
+        end
+      end
+
+      private
+
+      def elections
+        @elections ||= reorder(search.result)
+      end
+
+      def election
+        @election ||= elections.find_by(id: params[:id])
```

Comment:
If you would use plain Find, you do not need line 17 in this file.

```suggestion
        @election ||= elections.find(params[:id])
```

Reply:
Oh, I see